_includes/head.html

```diff
@@ -8,7 +8,7 @@
 
   {% assign custom_url = site.url | append: site.baseurl %}
   {% assign full_base_url = custom_url | default: site.github.url %}
-  <link rel="stylesheet" href="{{ "/css/main.css" | prepend: full_base_url }}">
+  <link rel="stylesheet" href="{{ "/assets/css/main.css" | prepend: full_base_url }}">
```

Comment:
For context, the idea here is that `/assets` will be a magic folder that will be included in the downstream site, saving the user from needing to create a dedicated `scss` file. See https://github.com/jekyll/minima/issues/16.

For existing users, it should "just work", but will need to require Jekyll `~> 3.3` in order to ensure we're not breaking existing sites.